[ROX-11918]: Add custom route to central configuration (#48792)
openshift/openshift-docs: modules/central-configuration-options-operator.adoc

|`central.exposure.route.enabled`
|Set this to `true` to expose Central through an OpenShift route. The default value is `false`.

|`central.exposure.route.host`

> |Specify a custom host name to use for Central's route. Leave this blank to accept the default value from OpenShift.

Reviewer: ```suggestion
|Specify a custom host name to use for Central's route. Leave this unset to accept the default value from OpenShift.
```
Not sure how a route would behave if host is set to an empty string (as opposed to unset), but better safe than sorry.

> |Specify a custom host name to use for Central's route. Leave this unset to accept the default value from OpenShift.

Reviewer 2: ```suggestion
|Specify a custom hostname to use for Central's route. Leave this unset to accept the default value that {ocp} provides.
```
